cliquet/storage/redis.py

```diff
@@ -92,7 +91,7 @@ def create(self, resource, user_id, record):
         self.check_unicity(resource, user_id, record)
 
         record = record.copy()
-        _id = record[resource.id_field] = self.id_generator()
+        _id = record[resource.id_field] = resource.id_generator()
```

Comment:
I like how we make the backend use the resource class for specific operations, it's a really nice extension point.